src/android/SecureStorage.java

```diff
@@ -329,10 +552,10 @@ public void onActivityResult(int requestCode, int resultCode, Intent intent) {
         String service = request.getService();
         CallbackContext callbackContext = request.getCallbackContext();
 
-        handleCompletedRequest(type, service, callbackContext);
+        handleCompletedRequest(type, service, callbackContext, intent);
     }
 
-    private void handleCompletedRequest(IntentRequestType type, String service, CallbackContext callbackContext) {
+    private void handleCompletedRequest(IntentRequestType type, String service, CallbackContext callbackContext, Intent intent) {
```

Comment:
Does handleCompletedRequest require the intent? I couldn't find any usage, seems like a leftover of some experiences.